spec/spec_helper.rb

```diff
@@ -5,6 +5,11 @@
 
 require 'dotiw'
 
+if defined?(ActionView)
+  require 'action_pack'
+  require 'action_controller'
```

Comment:
Necessary so I could reference `ActionController::Base` in `spec/lib/dotiw_spec.rb`

Reply:
Shouldn't need it with Rails loaded if you use a Gemfile from gemfiles/rails...

Reply:
If I don't require `action_controller`, then I get the following error when running `rspec`

```
Failures:

  1) A better distance_of_time_in_words with output options ActionView should be callable via ActionController::Base.helpers
     Failure/Error: actual = ActionController::Base.helpers.distance_of_time_in_words(START_TIME, end_time, true, { vague: false })

     NameError:
       uninitialized constant ActionController
     # ./spec/lib/dotiw_spec.rb:334:in `block (4 levels) in <top (required)>'
```

Reply:
That makes sense as I think Rails loads it only when used. We should require it in the spec.

```ruby
describe 'A better distance_of_time_in_words' do
  if defined?(ActionView)
    include ActionView::Helpers::DateHelper
    include ActionView::Helpers::TextHelper
    include ActionView::Helpers::NumberHelper

    require 'action_controller'
  else
    include DOTIW::Methods
  end
```